### DISCUSSION TITLE
CRTK interface message types

### DISCUSSION THREAD

**Discussion by kschwan** [surgical-robotics-ai/surgical_robotics_challenge#35] [General]
A while ago I suggested changing the CRTK Cartesian position type from `TransformStamped` to `PoseStamped` (see collaborative-robotics/documentation#1). This change is now effective ([see here](https://github.com/collaborative-robotics/documentation/wiki/Robot-API-motion#measured_cp-measured-cartesian-position)). It makes it easy to visualize poses in RViz which I think is a nice debugging help.

Are we allowed to use a modified [launch_crtk_interface.py](https://github.com/collaborative-robotics/surgical_robotics_challenge/blob/master/scripts/surgical_robotics_challenge/launch_crtk_interface.py)? I could submit a pull request with the alternative CRTK interface if that helps.

**Comment by adnanmunawar** [comment]
I saw that discussion and I agree with your proposition. However, I am a little nervous about changing the message types this close to the challenge. Let me have an internal discussion on this and I shall get back to you :)

**Reply by kschwan** [reply]
OK, otherwise we'll manage just fine 🙂

I'd suggest just making a separate CRTK interface script as to not change the existing one if someone is using that.

**Comment by adnanmunawar** [comment]
Hi Kim, let's do it :D. Since you offered, can you please create a PR that changes the instances of `geometry_msgs/TransformStamped` to `geometry_msgs/PoseStamped`?

**Reply by kschwan** [reply]
Cool 😎
I can make a pull request tomorrow. Would you prefer a separate script or changing the one that's already there?

**Reply by adnanmunawar** [reply]
Let's change the scripts already there. Better to pull the plug on the old CRTK standard :)
